advanced/helm/integrator-with-analytics/integrator-with-analytics/templates/integrator-server-dashboard-deployment.yaml

```diff
@@ -12,56 +12,140 @@
   strategy:
     type: Recreate
+  selector:
+    matchLabels:
+      deployment: wso2ei-with-analytics-dashboard
   template:
     metadata:
       labels:
         deployment: wso2ei-with-analytics-dashboard
     spec:
+      initContainers:
+      - name: init-is-db
+        image: busybox:1.31
+        command: ['sh', '-c', 'echo -e "Checking for the availability of MySQL Server deployment"; while ! nc -z {{ default "wso2ei-rdbms-service-mysql" .Values.wso2.mysql.host }} 3306; do sleep 1; printf "-"; done; echo -e "  >> MySQL Server has started";']
+      {{ if .Values.wso2.centralizedLogging.enabled }}
+      - name: init-logstash-plugins-install
+        image: docker.elastic.co/logstash/logstash:{{ default "7.2.0" .Values.wso2.centralizedLogging.logstash.imageTag }}
+        command:
+          - /bin/bash
+          - "-c"
+          - |
+            set -e
+            ./bin/logstash-plugin install logstash-codec-multiline logstash-filter-grok
+        volumeMounts:
+          - name: shared-plugins
+            mountPath: /usr/share/logstash/plugins/
+      - name: init-elasticsearch
+        image: busybox:1.31
+        command: ['sh', '-c', 'echo -e "Checking for the availability of Elasticsearch Server deployment"; while ! nc -z {{ default "wso2-elasticsearch-master" .Values.wso2.centralizedLogging.logstash.elasticsearch.host }} 9200; do sleep 1; printf "-"; done; echo -e "  >> Elasticsearch server has started";']
+      {{ end }}
       containers:
       - name: wso2ei-with-analytics-dashboard
-        {{- if or (eq .Values.username "") (eq .Values.password "") }}
-        image: wso2/wso2ei-analytics-dashboard:6.5.0
-        {{- else }}
-        image: docker.wso2.com/wso2ei-analytics-dashboard:6.5.0
-        {{- end }}
-        imagePullPolicy: Always
-        ports:
-        -
-          containerPort: 9643
-          protocol: "TCP"
+        {{ if .Values.wso2.dockerRegistry }}
+        image: {{ .Values.wso2.dockerRegistry }}/{{ default "wso2ei-analytics-dashboard" .Values.wso2.deployment.wso2eiAnalyticsDashbaord.imageName }}:{{ .Values.wso2.deployment.wso2eiAnalyticsDashbaord.imageTag }}
+        {{ else if or (eq .Values.wso2.subscription.username "") (eq .Values.wso2.subscription.password "") }}
+        image: wso2/{{ default "wso2ei-analytics-dashboard" .Values.wso2.deployment.wso2eiAnalyticsDashbaord.imageName }}:{{ .Values.wso2.deployment.wso2eiAnalyticsDashbaord.imageTag }}
+        {{ else }}
+        image: docker.wso2.com/{{ default "wso2ei-analytics-dashboard" .Values.wso2.deployment.wso2eiAnalyticsDashbaord.imageName }}:{{ .Values.wso2.deployment.wso2eiAnalyticsDashbaord.imageTag }}
+        {{ end }}
         livenessProbe:
           exec:
             command:
-            - /bin/sh
-            - -c
-            - nc -z localhost 9643
+              - /bin/bash
+              - -c
+              - nc -z localhost 9443
```

Comment:
@Savidude as per my understanding the port should be 9643 rather than 9443.

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/180/commits/4eab0894505a022e409f632e44196a919575ceb1